CHANGES.rst

```diff
@@ -313,6 +313,7 @@ Testing
   As a side-effect, this removes the need for test server lockfiles,
   meaning that any remaining ``tests/test-server.lock`` files can safely be
   deleted.
+* karma: refactor HTML search tests to use fixtures generated by Sphinx.
```

Comment:
This needs to be moved up to the current release. Don't forget to credit yourself!